garrysmod/lua/postprocess/stereoscopy.lua

```diff
@@ -15,7 +15,7 @@ function RenderStereoscopy( ViewOrigin, ViewAngles )
 	local w = ScrW() / 2.2
 	local h = ScrH() / 2.2
 
-	local Right = ViewAngles:Right() * pp_stereoscopy_size:GetFloat()
+	local Right = ViewAngles:Right() * math.Clamp( pp_stereoscopy_size:GetFloat(), 0, 10 )

```

Comment:
Perhaps this should reference a new convar (with FCVAR_CHEAT of course) instead of hardcoding the min/max.

Reply:
Why not write min/max into the convar creation above?

`local pp_stereoscopy_size = CreateClientConVar( "pp_stereoscopy_size", "6", true, false, "", 0, 10 )`

Reply:
Yeah, pp_stereoscopy_min and pp_stereoscopy_max will suffice. It will have to scale the AddControl call below as well, though.

Reply:
Declaring min and max value in CreateClientConVar won't clamp the values if they have been set before introducing those ranges...